### DISCUSSION TITLE
¿Cómo gestionar sitios con subdominios?

### DISCUSSION THREAD

**Discussion by palmerabollo** [PucelaBits/websegura#29] [General]
Hay sitios, como la Diputación de Valladolid, que tienen un dominio con varios subdominios. Los subdominios usan un certificado wildcard en ocasiones (por ejemplo, *.diputaciondevalladolid.es en este caso), por lo que la información de cada uno de los subdominios suele ser redundante. ¿Se os ocurre alguna forma de mejorar esa parte? Ocurre lo mismo en la página del Ayuntamiento, etc.

![Screenshot 2021-01-25 at 16 27 07](https://user-images.githubusercontent.com/673420/105726461-67393680-5f2a-11eb-9163-52e40ac0ba7e.png)

**Comment by nukeador** [comment]
Perdón que no había visto esto.

Creo que cada subdominio puede estar en una máquina diferente y por lo tanto tener una nota diferente, por lo que habría que añadirlos uno a uno, ¿no?

**Comment by XtoreX** [comment]
Coincido en añadirlos uno a uno, pues es cierto que en ocasiones están en diferentes máquinas.

Lo que igual se podría hacer, es mostrar primero el dominio principal, y a continuación, si la valoración es la misma no mostrar el subdominio correspondiente, o bien incluirlo en la página de resultados del dominio principal. En cualquier caso, si fuese inferior, sí se mostraría.

De esta manera, se podría intentar reducir un poco el tamaño de los listados.

¿Os parece muy rebuscado?

**Reply by palmerabollo** [reply]
Algún tipo de agrupación visual / inteligencia a nivel de UI podría estar bien, sí.
